common/src/sql/dbinit.sql

```diff
@@ -890,6 +892,7 @@ CREATE UNIQUE INDEX on omicron.public.image (
 ) WHERE
     time_deleted is NULL;
```

Comment:
Do you also need an index on `is_global` for the filter by `project_id=id OR is_global = true`?

Reply:
Yeah, that's a good point.